src/web/app/components/copy-course-modal/copy-course-modal.component.ts

```diff
@@ -105,6 +107,8 @@ export class CopyCourseModalComponent implements OnInit {
    * Clears all selected feedback sessions.
    */
   clearSelectedFeedbackSession(): void {
-    this.chosenFeedbackSessions = new Set<FeedbackSession>();
+    this.chosenFeedbackSessions.clear();
   }
+
+  private zeroPad: any = (num: number) => String(num).padStart(2, '0');
```

Comment:
Am wondering if this can be outside the class? It is also fine being inside too:p